presto/dataset.py

```diff
@@ -216,8 +216,8 @@ class WorldCerealInferenceDataset(Dataset):
         "temperature-mean": "temperature_2m",
     }
 
-    def __init__(self):
-        self.path_to_files = data_dir / "inference_areas"
+    def __init__(self, input_dir: Optional[Union[str, Path]] = None):
+        self.path_to_files = data_dir / "inference_areas" if input_dir is None else Path(input_dir)
```

Comment:
Instead of making `input_dir` optionally `None`, can you just set the default argument to be `data_dir`?

Reply:
does this work?:
```python
def __init__(self, input_dir: Path = data_dir / "inference_areas"):
```

Reply:
Apparently it does!